mvc-api/mvc-service/app/managers/EventObjectProducer.java

```diff
@@ -0,0 +1,31 @@
+package managers;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.sunbird.search.util.SearchConstants;
+
+import java.util.UUID;
+
+public class EventObjectProducer {
+    EventProducer eventProducer = new EventProducer();
+    public void addToEventObj(JSONObject content, String contentId) throws ParseException {
+        // Add label
+        content.put("label","MVC");
```

Comment:
Please don't use `label` as metadata to identify an MVC content. This name is too generic.

Reply:
label is the sole separator decided and we are using this everywhere
please discuss some parameter with sumit if this needs a change